meson.build

```diff
@@ -15,7 +15,12 @@ extra_defines = ['-DMIN_CPP_VERSION=201703L']
 cc = meson.get_compiler('c')
 cxx = meson.get_compiler('cpp')
 
+tizen_6_0 = false
```

Comment:
Suggestion:

Add a meson_options with ```tflite_package_name``` with the default value of ```tensorflow2-lite```. And let ```tflite_dep = dependency('tensorflow-lite', required: false)``` use this option variable instead.

Then, with the minimal changes, you can control this at .spec file.

This removes the knowledge dependency on Tizen versions from meson code, reducing inter-module coupling! (better design)

Reply:
this will be removed after tensorflow2 spec in tizen 6.0 is fixed.